src/main/java/org/folio/fqm/repository/EntityTypeRepository.java

```diff
@@ -227,6 +227,7 @@ private EntityTypeColumn handleBooleanCustomField(String id,
       .valueGetter(String.format(STRING_EXTRACTOR, sourceViewExtractor, refId))
       .labelAlias(name)
       .queryable(true)
+      .essential(true)
```

Comment:
It is necessary for custom fields declared in simple entity types to be visible in composite ones. In this case, we can filter in the UI by custom fields from both Order and Order Line entities.